### DISCUSSION TITLE
Audio not included in .mkv file

### DISCUSSION THREAD

**Discussion by Philius** [Motion-Project/motionplus#45] [Q&A]
### Did you read the guide?

Yes

### What is the base version number of MotionPlus being used?

0.1.x

### What was the install method?

Built from source code

### What is base architecture?

x86/x64

### What is the distro being used?

Debian

### Disto version number

12

### Camera/Sound type(s) being used?

v4l2 camera, Sound via PulseAudio

### Describe the issue/problem and steps to reproduce

I'm using a USB camera, with integrated microphone.
The video device was /dev/video0 as it's a v4l2 device.
In camera1.conf I have `v4l2_device /dev/video0`.
I used `pactl list short sources` to get the name of the sound device.
The audio triggers from sound1.conf
```
snd_device alsa_input.usb-HD_WEB_CAMERA_HD_WEB_CAMERA_SN0001-02.mono-fallback
snd_params source=pulse
snd_show on
snd_alerts alert_id=01,volume_level=01,volume_count=10,freq_low=100,freq_high=3399,alert_nm=alert
```
Show up in the log.
I've tested the device with
`ffmpeg -f pulse -i 'alsa_input.usb-HD_WEB_CAMERA_HD_WEB_CAMERA_SN0001-02.mono-fallback' -ac 1 recording.m4a
`
and it works, as verified with  `ffplay recording.m4a`.
But the recorded .mkv file is missing audio.

### Relevant MotionPlus log output (at log_level 8)

```shell
Mar 20 17:50:05 [INF][ALL][01:sl01:MyMic1] snd_check_alerts: Sound Alert 1-alert : level 1 count 2015 max vol 81
```

**Comment by Mr-Dave** [comment]
v4l2 devices are not those that are supported for recording audio.  The docs say certain cameras and those are currently rtsp cameras with passthrough specified.

The sound device / alert is for frequency detection.

**Reply by Philius** [reply]
I added the sound alert to show that the device is being used correctly.

**Comment by Philius** [comment]
The audio input is a pulseaudio device.
`snd_params source=pulse`
Are those supported?

Note that my goal is to use the raspberry pi camera with a USB mic, testing on a PC is just easier.
Is the `raspberry pi camera` + `USB mic` combination supported?
